src/savedEditor.ts

```diff
@@ -28,9 +30,26 @@ export class SavedEditor {
             preview: false
         };
 
+        let cfg = workspace.getConfiguration().get<IConfig>(ExtensionKey);
+        if (cfg === undefined) {
+            cfg = defaultIConfig;
+        }
+        let cfgTerminalFocus = cfg.returnFocusOnTerminal
```

Comment:
Re-use the variable name via destructuring:

```suggestion
        let { returnFocusOnTerminal } = cfg;
```